package.json

```diff
@@ -11,6 +11,7 @@
     "web-ui": "webpack --config components/webpack/webpack.config.js --progress --colors",
     "build-storybook": "build-storybook -c .storybook -o .storybook-out",
     "storybook": "start-storybook",
+    "build": "npm run build-storybook",
```

Comment:
This command name is a bit too generic, since this isn't the main output of this repo. Please can we call it something like `storybook-build` or `storybook:build`?